.github/workflows/update_license.yml

```diff
@@ -0,0 +1,52 @@
+name: Update license file
+
+on:
+  push:
+    branches:
+      - 'dependabot/**'
+
+# Workflows triggered by Dependabot have a read-only token by default.
+# Need to grant write permissions so this workflow can commit changes to the license file.
+permissions:
+  contents: write
+
+jobs:
+  update_license_file:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version:  '17.0.4+8'
+          distribution: 'adopt'
+
+      - name: install git secrets
+        run: |
+          wget --no-verbose -O git-secrets-1.3.0.tar.gz https://github.com/awslabs/git-secrets/archive/1.3.0.tar.gz
+          tar -zxf git-secrets-1.3.0.tar.gz
+          cd git-secrets-1.3.0
+          sudo make install   
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Build with mvnw
+        run: ./mvnw clean install
+
+      - name: Push changes if license file changed
+        run: |
+          if [ -n "$(git status --porcelain | grep "THIRD-PARTY-LICENSES.txt")" ]; then
+            git config user.name "github-actions[bot]"
+            git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Out of curiosity, why this particular email address?

Reply:
Got it from https://github.com/dockstore/dockstore-ui2/pull/1517 which seems to be a github actions thing
https://github.com/orgs/community/discussions/26560